pallets/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -349,7 +349,7 @@ where
 			nobody.unwrap(),
 			contract_id,
 			0u32.into(),
-			Weight::from_ref_time(100_000_000_000),
+			Weight::from_ref_time(100_000_000_000).set_proof_size(u64::MAX),
```

Comment:
Why?

Reply:
Provided [details](https://github.com/Cerebellum-Network/blockchain-node/pull/57#issue-1793846547) in the description of Pull-Request.